const n1 = 20;
const n2 = 30;

console.log(n1);
console.log(n2);
try{
    console.log(n3);
} catch(error){
    console.log(error);
}
